### PR TITLE
docs: quickstart for radondb mysql operator #127

### LINKED ISSUE #127: [documentation] Deployment document of operator version
**Is your feature request related to a problem? Please describe.**

TODO:
- deploy radondb mysql opertor on k8s
- deploy radondb mysql opertor on k8e

**Describe the solution you'd like**

reference: [deploy_radondb-mysql_on_kubernetes](https://github.com/radondb/radondb-mysql-kubernetes/blob/main/docs/Kubernetes/deploy_radondb-mysql_on_kubernetes.md)

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by runkecheng**
### What type of PR is this?

quickstart for radondb mysql operator

### Which issue(s) this PR fixes?

Fixes #127

### CODE REVIEW

File: docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+
+## 部署准备
+
+* 已准备可用 Kubernetes 集群。
+
+## 部署步骤
+
+### 步骤 1: 克隆代码
+
+```github
```
Comment (hustjieke) [inline docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md:41]: remove `github`

File: docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+git clone https://github.com/radondb/radondb-mysql-kubernetes.git
+```
+
+### 步骤 2: 部署 Operator
+
+```helm
```
Comment (hustjieke) [inline docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md:47]: changed to :

`For Helm V3，使用说明：`
```
helm install [NAME] [CHART] [flags]
```

File: docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+<For Helm V3>
+helm install <release> radondb-mysql-kubernetes/charts/mysql-operator
+```
+
+以下指定 release 名为 `demo` , 创建一个名为 `demo-mysql-operator` 的 [Deployment](#7-deployments)，以及创建一个名为 `clusters.mysql.radondb.com` 的 [CRD](#8-CRD)。
+
+```helm
```
Comment (hustjieke) [inline docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md:54]: remove `helm` and `<For Helm V3>`

File: docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+<For Helm V3>
+helm install demo radondb-mysql-kubernetes/charts/mysql-operator
```
Comment (hustjieke) [inline docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md:56]: add `helm list` after install success, like: 
```
hustjason:gravity gaoriyao$ helm list
NAME     	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART              	APP VERSION
demo     	default  	1       	2021-05-18 11:44:35.793793 +0800 CST	deployed	radondb-mysql-1.0.0	5.7.33     
testmysql	default  	1       	2021-05-18 09:37:38.548275 +0800 CST	deployed	mysql-8.5.8        	8.0.25     
```

Comment (runkecheng) [reply docs/Kubernetes/deploy_radondb-mysql_operator_on_k8s.md:60]: In this way, it seems a bit redundant, which one is better?

**Review by andyli029 — COMMENTED** [review]
LGTM.

**Pull request merged**